tests/core/test_configuration.py

```diff
@@ -703,5 +703,20 @@ def test_enum_default_task_weight_rule_from_conf(self):
         with pytest.raises(AirflowConfigException) as ctx:
             test_conf.validate()
         exception = str(ctx.value)
-        message = "sidestream is not an accepted config for [core] default_task_weight_rule"
+        message = (
+            "[core] default_task_weight_rule` should not be 'sidestream'. Possible values: "
+            "upstream, downstream, absolute"
```

Comment:
Displaying the available options will help us troubleshoot configuration problems.